composer.json

```diff
@@ -6,46 +6,44 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": ">=7.3",
+        "php": ">=7.4",
         "drupal/allowed_formats": "^1.4",
-        "drupal/core": "^8.9 || ^9.1",
+        "drupal/core": "^9.2",
         "drupal/paragraphs": "^1.12",
         "drupal/typed_link": "^2.0"
     },
     "require-dev": {
+        "behat/behat": "^3.10",
+        "behat/mink-extension": "^2.3.1",
         "composer/installers": "^1.5",
-        "consolidation/annotated-command": "^2.8.2",
+        "cweagans/composer-patches": "^1.7",
```

Comment:
Why do we introduce `cweagans/composer-patches` this package here. We have have reason, it would be nice to put more information in `_readme`.